gammapy/irf/energy_dispersion.py

```diff
@@ -541,6 +545,291 @@ def gauss_energy_dispersion_matrix(ebounds, sigma=0.2):
+        plt.xlim(0, 4)
+
+
+    Plot evolution of bias and resolution as a function of true energy
+    for a given offset
+
+    .. plot::
+        :include-source:
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from gammapy.irf import EnergyDispersion2D
+        from gammapy.datasets import load_edisp2D_fits_table
+        from gammapy.spectrum.energy import Energy
+        from astropy.coordinates import Angle
+        edisp2D = EnergyDispersion2D.from_fits(load_edisp2D_fits_table())
+        migra = np.linspace(0.1,2,80)
+        energy = Energy.equal_log_spacing(0.13,60,60,'TeV')
+        offset = Angle([0.554], 'deg')
+        edisp2D.plot_bias(offset=offset, energy=energy, migra=migra)
+        plt.xscale('log')
+
+    Write RMF fits table
+
+
+    """
+
+    def __init__(self, etrue_lo, etrue_hi, migra_lo, migra_hi, offset_lo,
+                 offset_hi, dispersion):
+
+        if not isinstance(etrue_lo, Quantity) or not isinstance(etrue_hi, Quantity):
+            raise ValueError("Energies must be Quantity objects.")
+        if not isinstance(offset_lo, Angle) or not isinstance(offset_hi, Angle):
+            raise ValueError("Offsets must be Angle objects.")
+
+        self.migra_lo = migra_lo
+        self.migra_hi = migra_hi
+        self.offset_lo = offset_lo
+        self.offset_hi = offset_hi
+        self.dispersion = dispersion
+
+        self.ebounds = EnergyBounds.from_lower_and_upper_bounds(etrue_lo, etrue_hi)
+        self.energy = self.ebounds.log_centers
+        self.offset = (offset_hi + offset_lo) / 2
+        self.migra = (migra_hi + migra_lo) / 2
+
+        self._prepare_linear_interpolator()
+
+    @classmethod
+    def from_fits(cls, hdu):
+        """Create `EnergyDispersion2D` from ``GCTAEdisp2D`` format HDU.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            ``ENERGY DISPERSION`` extension.
+            
+        """
+
+        data = hdu.data
+        header = hdu.header
+        e_lo = EnergyBounds(data['ETRUE_LO'].squeeze(), header['TUNIT1'])
+        e_hi = EnergyBounds(data['ETRUE_HI'].squeeze(), header['TUNIT2'])
+        o_lo = Angle(data['THETA_LO'].squeeze(), header['TUNIT5'])
+        o_hi = Angle(data['THETA_HI'].squeeze(), header['TUNIT6'])
+        m_lo = data['MIGRA_LO'].squeeze()
+        m_hi = data['MIGRA_HI'].squeeze()
+        matrix = data['MATRIX'].squeeze()
+
+        return cls(e_lo, e_hi, m_lo, m_hi, o_lo, o_hi, matrix)
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyDispersion2D` from ``GCTAEdisp2D`` format FITS file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdulist = fits.open(filename)
+        return cls.from_fits(hdulist['ENERGY DISPERSION'])
+
+    def evaluate(self, offset=None, e_true=None, migra=None):
+        """Probability for a given offset, true energy, and migration
```

Comment:
There's the option to use more explicit names like `dp_dy`, but it's still cryptic, so I don't have a proposal for something better at the moment.

Adding a formula to the docs to better explain what the functions return (i.e. which probability density) is easy.
See [here](https://gammapy.readthedocs.org/en/latest/api/gammapy.irf.TablePSF.html#gammapy.irf.TablePSF.evaluate) as an example.